main.go

```diff
@@ -1,11 +1,29 @@
 package main
 
-import "fmt"
-
+import (
+	"fmt"
+	"strconv"
+)
 
 func main() {
+
+	var status string = "IN PROGRESS"
+	var createdTasks int = 25
+	var isCompleted bool = false
+	const availableTasks int = 100
+	const (
+		low = iota
+		medium
+		high
+	)
```

Comment:
In this case you can store constants in global scope. Functional scope is not reachable outside of the scope, but those values logically can be re-usable. So, makes sense to use globally